### ISSUE TITLE
Support Svelte 5

### ISSUE THREAD

**Issue by Shaokun-X** [NatoBoram/heroicons.svelte#110]
Hey, since [Svelte 5 has become alive](https://svelte.dev/blog/svelte-5-is-alive), is there any plan to support it?

**Comment by NatoBoram** [comment]
Yes! I was waiting first for the docs to officially release since they were late, now I just need to find the time to update it
